### PR TITLE
Add .action file for execution trajectory in a ROS action

### PR DESCRIPTION
**Pull request by wkentaro**
For ros-planning/moveit_ros#719
Being moved from #24

### CODE REVIEW

**Comment by v4hn** [comment]
-1 on the name.
I agree with @davetcoleman's [comment](https://github.com/ros-planning/moveit_ros/pull/719#discussion_r71153856).
Please rename the action to "ExecutePath.action".

**Comment by wkentaro** [comment]
Fixed.

File: action/ExecutePath.action [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+---
+
+# Error code - encodes the overall reason for failure
+MoveItErrorCodes error_code
+
+# The full starting state of the robot at the start of the trajectory
+moveit_msgs/RobotState trajectory_start
+
+# The trace of the trajectory recorded during execution
```
Comment (davetcoleman) [inline action/ExecutePath.action]: could you explain "trace" better... in what cases would it differ from the input ``trajectory`` and at what discretization level are you expecting this returned trajectory to be at?

**Comment by davetcoleman** [comment]
I'm not sure if they are useful or not, but it wasn't obvious to me what they would be used for. Seeing how the old ``ExecuteKnownTrajectory.srv`` doesn't have anything similar, it makes sense to not have them

**Comment by wkentaro** [comment]
I agree. I think the results can be added afterward when it is required.

**Comment by davetcoleman** [comment]
cherry-picked to J

**Comment by wkentaro** [comment]
Thanks!

**Comment by rhaschke** [comment]
> Please rename the action to `ExecutePath.action`.

Hm. Even better would have been `ExecuteTrajectory.action`. All other stuff related to joint-space trajectories is called like this. Also, a `RobotTrajectory` is passed in the message...

@v4hn @davetcoleman: What do you think?

Should we also rename the existing `ExecuteKnownTrajectory.srv` to `ExecuteTrajectory.srv` for unification?

**Comment by davetcoleman** [comment]
I see your point, trajectory makes more sense than path. +1

I don't see nearly enough advantage to change the long existing service
name though.

**Comment by 130s** [comment]
Are we going to make a change discussed after this PR was merged any sooner?
I like to fix https://github.com/ros-planning/moveit_msgs/commit/c5c4c6181e3e9a24ac1f8020e1ad3dbd05e13b00#commitcomment-18705574 and make a release asap (within a day or so), so it'd be great if both changes can be released together.

**Comment by rhaschke** [comment]
@130s Name fixed with #27.

**Pull request merged**